contractcourt/channel_arbitrator_test.go

```diff
@@ -704,6 +718,7 @@ func TestChannelArbitratorLocalForceClosePendingHtlc(t *testing.T) {
 
 	// The force close request should trigger broadcast of the commitment
 	// transaction.
+	arbLog := chanArb.log.(*testArbLog)
 	assertStateTransitions(
 		t, arbLog.newStates, StateBroadcastCommit,
 		StateCommitmentBroadcasted,
```

Comment:
These are nice to implement as methods on the context, especially if the context knows `t` as well.

Reply:
Ported them over along with `assertState`.